lib/metasploit/framework/data_service/remote/http/response_data_helper.rb

```diff
@@ -45,6 +73,60 @@ def process_file(base64_file, save_path)
     save_path
   end
 
+  # Converts a Hash or JSON string to an ActiveRecord object.
+  # Importantly, this retains associated objects if they are in the JSON string.
+  #
+  # Modified from https://github.com/swdyh/toar/
+  # Credit to https://github.com/swdyh
```

Comment:
Does the `LICENSE` file need to be updated because of this?

Reply:
I'm not sure. This repo that I'm pulling form does not have a LICENSE attached with it.

Reply:
TODO: backlog item to determine how to address, and contact author for their license information.